<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="2082" started-at="2017-08-26T10:05:45Z" finished-at="2017-08-26T10:05:47Z">
    <groups>
    </groups>
    <test name="api" duration-ms="2082" started-at="2017-08-26T10:05:45Z" finished-at="2017-08-26T10:05:47Z">
      <class name="test.DebugTestNG">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:test.DebugTestNG@8bd1b6a]" name="beforeClass" is-config="true" duration-ms="60" started-at="2017-08-26T18:05:45Z" finished-at="2017-08-26T18:05:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:test.DebugTestNG@8bd1b6a]" name="beforeMethod" is-config="true" duration-ms="1122" started-at="2017-08-26T18:05:45Z" finished-at="2017-08-26T18:05:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="exchangeGift()[pri:0, instance:test.DebugTestNG@8bd1b6a]" name="exchangeGift" duration-ms="883" started-at="2017-08-26T18:05:46Z" finished-at="2017-08-26T18:05:47Z">
          <exception class="net.sf.json.JSONException">
            <message>
              <![CDATA[JSONObject["prepayId"] not found.]]>
            </message>
            <full-stacktrace>
              <![CDATA[net.sf.json.JSONException: JSONObject["prepayId"] not found.
	at net.sf.json.JSONObject.getString(JSONObject.java:1847)
	at test.DebugTestNG.exchangeGift(DebugTestNG.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- net.sf.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exchangeGift -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:test.DebugTestNG@8bd1b6a]" name="afterMethod" is-config="true" duration-ms="1" started-at="2017-08-26T18:05:47Z" finished-at="2017-08-26T18:05:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:test.DebugTestNG@8bd1b6a]" name="afterClass" is-config="true" duration-ms="0" started-at="2017-08-26T18:05:47Z" finished-at="2017-08-26T18:05:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- test.DebugTestNG -->
    </test> <!-- api -->
  </suite> <!-- Suite -->
</testng-results>
